tests/console/cryptsetup.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case 1528909  - FIPS: cryptsetup
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+
+    my $slef        = shift;
```

Comment:
`$slef`? funny :-) It's a typo but you never use `$slef` so better just delete the line or replace by `my $self = @_;`